priv/perf/apps/load_test/test/load_tests/runner/utxos_test.exs

```diff
@@ -0,0 +1,42 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule LoadTest.Runner.UtxosTest do
+  use ExUnit.Case
+
+  @moduletag :utxos
+
+  test "deposits test" do
+    token = "0x0000000000000000000000000000000000000000"
+    initial_amount = 760
+    fee = 75
+
+    config = %{
+      chain_config: %{
+        token: token,
+        initial_amount: initial_amount,
+        fee: fee
+      },
+      run_config: %{
+        tps: 1,
+        period_in_seconds: 20
+      },
+      timeout: :infinity
```

Comment:
why infinity??

Reply:
the default timeout is 10_000 ms for task execution inside of chaperon

```
2020-10-29 13:24:13.437 [error] ⋅Task #PID<0.500.0> started from #PID<0.501.0> terminating
** (stop) exited in: Task.await(%Task{owner: #PID<0.501.0>, pid: #PID<0.879.0>, ref: #Reference<0.3543989965.2161639427.128884>}, 10000)
    ** (EXIT) time out
    (elixir 1.10.2) lib/task.ex:607: Task.await/2
    (chaperon 0.3.1) lib/chaperon/session.ex:366: Chaperon.Session.await/3
    (elixir 1.10.2) lib/enum.ex:2111: Enum."-reduce/3-lists^foldl/2-0-"/3
    (chaperon 0.3.1) lib/chaperon/scenario.ex:198: Chaperon.Scenario.with_scenario/3
    (chaperon 0.3.1) lib/chaperon/scenario.ex:172: Chaperon.Scenario.run/2
    (chaperon 0.3.1) lib/chaperon/scenario.ex:122: Chaperon.Scenario.execute/2
    (elixir 1.10.2) lib/task/supervised.ex:90: Task.Supervised.invoke_mfa/2
    (elixir 1.10.2) lib/task/supervised.ex:35: Task.Supervised.reply/5
    (stdlib 3.12) proc_lib.erl:249: :proc_lib.init_p_do_apply/3
```